index.html

```diff
@@ -555,21 +555,21 @@ <h4>Prove Presentation</h4>
       </section>
 
       <section>
-        <h4>Presentation Availability</h4>
+        <h4>Initiate Exchange</h4>
```

Comment:
@jandrieu I waffled back and forth on "Initiate Presentation Exchange" and what I ended up using above. Any ideas? Is this what you were going for?

Reply:
Initiate exchange sounds pretty good to me